docs/draft/6-payloads.md

```diff
@@ -191,6 +195,22 @@ message ImageMessage {
 }
 ```
 
+##### Audio content type
+
+A `ChatMessage` with `AUDIO` `Content/Type` MUST also specify the `payload` of the audio
+and the `type`
+
+```protobuf
+message AudioMessage {
+  bytes payload = 1;
+  AudioType type = 2;
+  enum AudioType {
+    UNKNOWN_AUDIO_TYPE = 0;
+    AAC = 1;
```

Comment:
Any security concerns?

@rajeevgopalakrishna

Reply:
Potential concerns:
1. Spam and network congestion given the size of audio msgs? Should the TTL be different for such msgs?
2. Any upper limit on the size of audio msgs?
3. Any impact on the PoW for audio msgs?
4. Is AAC the only supported format? What happens if a corrupt AAC file is used?

Anything else @status-im/security-internal?

Reply:
Thanks for the points.

1) Currently there's no difference between an audio file or a text message in terms of bandwidth. Anyone can fill up a text file with 0s and 1s and send it through the network and it will be received by the users. This change in the specs does not change the maximum size limit.
2) We are not going to limit the size here, currently any envelope in waku is capped to 1MB, so it's implicitly capped for now. If there's a need to send larger files we can address that at a different layer, by splitting the whole protobuf (encoded) into chunks.
3) No, PoW would not be impacted, we lowered it for images already, but effectively this allows only to send a different type of content, but does not alter the quantity of data transmitted
4) I have added AMR as well, the protocol won't be checking that the file is properly formatted, I would leave that responsibility to the actual player, so that would be the clients responsibility to ensure that corrupted files are handled appropriately.

Reply:
Could you add the assumption that the client is expected to sanitize the file from the message? I think its good to express that in the spec.

Reply:
Sure, would you use exactly the same wording, something like:

`clients SHOULD sanitize the payload before accessing its content`

or rather there's some better way to phrase this?

Reply:
I think that's a MUST case. I'd suggest:

Clients MUST sanitize the payload before accessing its content. In particular, clients MUST verify that the payload size is constrained and a secure decoder is chosen on the receivers end.